src/server/src/main/java/io/cassandrareaper/storage/PostgresStorage.java

```diff
@@ -841,52 +841,54 @@ private boolean segmentIsWithinRanges(RepairSegment seg, List<RingRange> ranges)
   }
 
   @Override
-  public void storeMetric(GenericMetric metric) {
+  public void storeMetrics(List<GenericMetric> metrics) {
     if (null != jdbi) {
       try (Handle h = jdbi.open()) {
-        Instant metricTs = Instant.ofEpochMilli(metric.getTs().getMillis());
-        IStoragePostgreSql storage = getPostgresStorage(h);
-        int rowsUpdated = storage.updateMetricSourceNodeTimestamp(
-            metric.getClusterName(),
-            metric.getHost(),
-            metricTs
-        );
-        if (rowsUpdated == 0) {
+        for (GenericMetric metric:metrics) {
```

Comment:
From the description I thought this PR just involved changes with writes to the Cassandra backend. Can you explain these changes? (mostly because I don't understand 😄)

Reply:
Yes, it's the case but in order to achieve this, I had to modify the signature of a method from an interface that is also used by the postgres backend class.
Instead of storing metrics one by one, I know provide a list of metrics which will be batched in Cassandra but inserted one by one in Postgres.